src/sys/socket/consts.rs

```diff
@@ -9,6 +9,7 @@ mod os {
     pub const AF_INET: c_int  = 2;
     pub const AF_INET6: c_int = 10;
     pub const AF_NETLINK: c_int = 16;
+    pub const AF_NETLINK: c_int = 17;
```

Comment:
Oops, I had to copy/paste the patch and I fat fingered. I will fix shortly

On Tuesday, February 23, 2016, Paul Osborne <notifications@github.com>
wrote:

> In src/sys/socket/consts.rs
> <https://github.com/nix-rust/nix/pull/278#discussion_r53801403>:
>
> > @@ -9,6 +9,7 @@ mod os {
> >      pub const AF_INET: c_int  = 2;
> >      pub const AF_INET6: c_int = 10;
> >      pub const AF_NETLINK: c_int = 16;
> > +    pub const AF_NETLINK: c_int = 17;
>
> Should be AF_PACKET
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nix-rust/nix/pull/278/files#r53801403>.
>